src/main/java/org/fundacionjala/coding/jimmy/Kata1.java

```diff
@@ -8,7 +8,7 @@
  */
 public class Kata1 {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)